web_scraping/browser.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+import concurrent.futures
+import requests
+import json
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from bs4 import BeautifulSoup
+import base64
+import asyncio
+import aiohttp
+
+chrome_options = Options()
+chrome_options.add_argument("--headless")
+
+directory = Path("output")
+
+async def get_html(url, session):
+    async with session.get(url) as response:
+        response.raise_for_status()
```

Comment:
Note that this function can cause an error if the status isn't 200 and you didn't handle that